app-modules/student-data-model/src/Filament/Resources/StudentResource/Pages/ViewStudent.php

```diff
@@ -165,9 +171,45 @@ public function infolist(Infolist $infolist): Infolist
 
     public function sisRefresh()
     {
+        [$olympusUrl, $olympusKey] = Landlord::execute(function (): array {
+            $settings = app(OlympusSettings::class);
+
+            return [
+                rtrim($settings->url, '/'),
+                $settings->key,
+            ];
+        });
```

Comment:
Maybe we could have an `Olympus` service similar to `ThesisElements` with a pre-configured `makeRequest()` method that creates a pending request using the `baseUrl()` and auth token. Then it's reusable for future sync requests

Reply:
Yeah I think that is a good idea. I think it can be a lot simpler than `ThesisElements` for now with just a static method. You can let me know what you think.

I think ultimately we will improve this to instead actually come out an "Integration" for this side. We may want to consider AdvisingApp to consider Olympus a SIS integration that could be swapped out